tox.ini

```diff
@@ -1,10 +1,48 @@
 [tox]
-envlist = py3-{lint,test}
+envlist =
+    fix,
+    lint,
+    test
 skipsdist = true
-sitepackages = true
+
+[base]
+deps =
+    -rrequirements.txt
```

Comment:
Shouldn't this be `requirements.txt` ?

Reply:
-r is the argument to read from the file given after. I agree, this is hard to read, you see the same e.g. here: https://github.com/openstack/ceilometer/blob/master/tox.ini#L10